src/components/CalculatorButton.js

```diff
@@ -0,0 +1,23 @@
+import './CalculatorButton.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- I would recommend not disabling the linter checks and fix the linter errors this file is raising.